app/src/main/java/com/poti/android/presentation/setting/SettingScreen.kt

```
@@ -0,0 +1,119 @@
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.poti.android.R
+import com.poti.android.core.designsystem.component.button.PotiMenuButton
+import com.poti.android.core.designsystem.component.display.PotiDivider
+import com.poti.android.core.designsystem.component.display.PotiDividerStyle
+import com.poti.android.core.designsystem.component.navigation.PotiHeaderPage
+import com.poti.android.core.designsystem.theme.PotiTheme
+
+@Composable
+fun SettingRoute(modifier: Modifier = Modifier) {
+}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)